src/main/kotlin/org/jetbrains/research/libsl/nodes/statements.kt

```diff
@@ -10,6 +10,56 @@ data class Assignment(
     override fun dumpToString(): String = "${left.dumpToString()} = ${value.dumpToString()};"
 }
 
+data class AssignmentWithCompoundOp(
+    val left: QualifiedAccess,
+    val op: CompoundOps,
+    val value: Expression
+) : Statement() {
+    override fun dumpToString(): String = "${left.dumpToString()} ${op.string} ${value.dumpToString()};"
+}
+
+data class AssignmentWithLeftUnaryOp(
+    val op: ArithmeticUnaryOp,
+    val value: Expression
+) : Statement() {
+    override fun dumpToString(): String = "${op.string}${value.dumpToString()};"
+}
+
+data class AssignmentWithRightUnaryOp(
+    val op: ArithmeticUnaryOp,
+    val value: Expression
+) : Statement() {
+    override fun dumpToString(): String = "${value.dumpToString()}${op.string};"
+}
+
+data class IfStatement(
+    val value: Expression,
+    val ifStatements: MutableList<Statement> = mutableListOf(),
+    val elseStatements: ElseStatement?
+) : Statement() {
+    override fun dumpToString(): String = buildString {
+        append("if${value.dumpToString()}")
+        appendLine(" {")
+        append(withIndent(formatListEmptyLineAtEndIfNeeded(ifStatements)))
+        appendLine("}")
+        elseStatements?.also {
+            append("else")
```

Comment:
Also, as well as this function prints `if` block and `else` blocks isn't a part of if (is has a different node class) this logic could be moved to the one next class